Squablr/Utils/RankingAlgorithmUtils.h

```diff
@@ -0,0 +1,18 @@
+//
+//  RankingAlgorithmUtils.h
+//  Squablr
+//
+//  Created by Zeke Reyes on 8/2/22.
+//8
+
+#import <Foundation/Foundation.h>
+#import "Profile.h"
+#import "Parse/Parse.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface RankingAlgorithmUtils : NSObject
++ (void) sortProfilesByCompatibility: (NSMutableArray *) profiles;
```

Comment:
I like to put a spaces in between the interface and end declaration lines and the body